CHANGELOG

```diff
@@ -1,3 +1,16 @@
+Version 1.1.0 (2021-07-??)
+--------------------------
+Run scalafmtCheckAll in github action (close #163)
+Migrate from travis to github actions (close #162)
+Add http4s module with a RegistryLookup (close #161)
+Bump kind-projector to 0.13.0 (close #160)
+Bump sbt to 1.5.4 (close #159)
+Bump scala compiler versions to 2.13.6 and 2.12.14 (close #158)
+Bump scala-collection-compat to 2.4.4 (close #157)
+Bump circe to 0.14.0 (close #156)
+Bump cats to 2.6.1 (close #155)
+Bump jackson to 2.10.5.1 (close #154)
```

Comment:
Does JSON validator also use 2.10.x? I think we can keep the minor version up to a validator's version.

Reply:
json-schema-validator 1.0.39 uses jackson 2.10.0.